configure

```diff
@@ -645,12 +647,12 @@ if test "$OPT_GETTEXT" != "no"; then
   if cc_check_headers "libintl.h"; then
     if cc_check_statement "libintl.h" "gettext(\"\");"; then
       GETTEXT_FOUND=found
-    elif cc_check_statement "libintl.h" "gettext(\"\");" "-lintl"; then
-      GETTEXT_LDFLAGS="-lintl"
-      GETTEXT_FOUND=found
```

Comment:
Please, provide an error you are fixing.

IMO, it's better to check first, for the gettext() function working without libiconv, and only if the check fails, then test with iconv. I suppose some dumb linkers can link libiconv (when -liconv provided), even when this library is actually unused.